h/emails/reply_notification.py

```diff
@@ -3,7 +3,7 @@
 from h import links
 
 
-def generate(request, notification):
+def generate(request, notification):  # pylint: disable=too-many-locals
```

Comment:
I think Pylint has a point here, but it's out of scope for this PR. I made another one here: https://github.com/hypothesis/h/pull/6841